pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+<!-- Please add the WIP label if the PR isn't complete. -->
+
+**Reviewer:** 
+
+<!-- Optional fields
+**CC:**
+**Depends on:** 
+-->
+
+## Description:
+
+Project task: 
+
+Relevant products which you have tested these changes against (i.e. extension, Windows, ...):
+- 
+
+Product PRs to test against:
+- 
+
+**Note: Upon merging this PR, you are responsible for updating the affected products to the latest version of this repo.**
+
+## Steps to test this PR:
```

Comment:
Not sure if that applies? The only step is "run tests see if they pass". The bigger concern is if test is correctly describing the edge-case that we are trying to capture.